### PR TITLE
created useCollection hook to fetch firestore collections

### PR DESCRIPTION
**Pull request by 13prashant**

### CODE REVIEW

File: src/pages/concerts/Concerts.js [hunk 1]
```diff
@@ -1,73 +1,20 @@
...
-      'Hetaxi Trivedi',
-      'Nidhi Adhvaryu',
-      'Prachi Shukla',
-      'Nishi Shukla',
-    ],
-  },
-];
-
 const Concerts = () => {
+  const { documents: concerts, isPending, error } = useCollection('concerts');
```
Comment (hridaysur) [inline src/pages/concerts/Concerts.js]: Can we please create a constant CollectionName class with 'concerts', 'lyrics' etc etc.. & Use this 'concerts' name in all the component from this class?

File: src/components/concerts/ConcertCard/ConcertCard.js [hunk 2]
```diff
@@ -24,37 +24,41 @@ const ITEM_HEIGHT = 48;
...
-            {concert.title}
+            {title}
           </Typography>
           <Typography variant="body1" color="text.primary">
-            {concert.venue}
+            {venue}
           </Typography>
           <Typography gutterBottom variant="body2" color="text.secondary">
-            {concert.time}
+            {new Date(time.toDate()).toDateString()}
```
Comment (hridaysur) [inline src/components/concerts/ConcertCard/ConcertCard.js]: Don't know the exact type of this time.toDate().. but I think we don't need to convert this in to new Date() again.. we should be able to directly convert this toDateString()..like time.toDate().toDateString() .. unless we need something specific in ToDateString().

File: src/pages/concerts/Concerts.js (hunk 1 above)
Comment (13prashant) [reply src/pages/concerts/Concerts.js]: Done

File: src/components/concerts/ConcertCard/ConcertCard.js (hunk 2 above)
Comment (13prashant) [reply src/components/concerts/ConcertCard/ConcertCard.js]: Done

**Review by hridaysur — APPROVED** [review]

**Pull request merged**